src/portable/stm/stm32f4/hal_stm32f4.c

```diff
@@ -0,0 +1,69 @@
+/**************************************************************************/
```

Comment:
Here, we should also rename the vendor from stm folder to **st** which is more accurate.